hack/preview.sh

```diff
@@ -32,24 +32,13 @@ if git rev-parse --verify $PREVIEW_BRANCH; then
 fi
 git checkout -b $PREVIEW_BRANCH
 
-# Set the domain for our rekor deployment.
-# We add the modified rekor.yaml file and this will get pushed to our preview branch.
-# This shouldn't ever be updated in $MY_GIT_BRANCH.
-# If you know a better way to make this magic happen, contact rnester@redhat.com
-domain=$( kubectl get ingresses.config.openshift.io cluster --template={{.spec.domain}} )
-echo
-echo "Setting rekor server domain to: $domain"
-echo
-sed -i "s/rekor-server.enterprise-contract-service.svc/rekor.$domain/" $ROOT/argo-cd-apps/base/enterprise-contract.yaml
-git add $ROOT/argo-cd-apps/base/enterprise-contract.yaml && git commit -m "Set domain for rekor"
```

Comment:
My understanding is that if we don't create a commit with this dynamic value then argocd will revert the change.

Reply:
The commit is later in this file, so we are creating one commit with all changes by preview.